### ISSUE TITLE
Tiến độ tiến hành chương Modern Convolutional Neural Networks

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#1283]
Mình hy vọng chương **Modern Convolutional Neural Networks** này (60 phần) chúng ta có thể hoàn thiện trước 14/06/2020.

## Giai đoạn 1 `38 parts`
* [x] 7.0 [Modern Convolutional Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `01 part`
* [x] 7.1. [Deep Convolutional Neural Networks (AlexNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+alexnet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `08 part`
* [x] 7.2. [Networks Using Blocks (VGG)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+vgg_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `03 part`
* [x] 7.3. [Network in Network (NiN)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+nin_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `03 part`
* [x] 7.4. [Networks with Parallel Concatenations (GoogLeNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+googlenet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `04 part`
* [x] 7.5. [Batch Normalization](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+batch-norm_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `08 part`
* [x] 7.6. [Residual Networks (ResNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+resnet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `06 part`
* [x] 7.7. [Densely Connected Networks (DenseNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+densenet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+1%22) `05 part`

## Giai đoạn 2 `22 parts`
* [x] 7.0 [Modern Convolutional Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `01 part`
* [x] 7.1. [Deep Convolutional Neural Networks (AlexNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+alexnet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `05 part`
* [x] 7.2. [Networks Using Blocks (VGG)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+vgg_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `02 part`
* [x] 7.3. [Network in Network (NiN)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+nin_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `02 part`
* [x] 7.4. [Networks with Parallel Concatenations (GoogLeNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+googlenet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `03 part`
* [x] 7.5. [Batch Normalization](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+batch-norm_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `04 part`
* [x] 7.6. [Residual Networks (ResNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+resnet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `02 part`
* [x] 7.7. [Densely Connected Networks (DenseNet)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+densenet_vn+label%3A%22chapter%3A+convolutional-modern%22+label%3A%22status%3A+phase+2%22) `03 part`
